src/Components/Aspire.Azure.Messaging.EventHubs/AzureMessagingEventHubsSettings.cs

```diff
@@ -46,6 +46,11 @@ public abstract class AzureMessagingEventHubsSettings : IConnectionStringSetting
     /// </summary>
     public TokenCredential? Credential { get; set; }
 
+    /// <summary>
+    /// Gets or sets a boolean value that indicates whether the health checks are disabled or not.
+    /// </summary>
```

Comment:
```suggestion
    /// <summary>
    /// Gets or sets a boolean value that indicates whether the Azure Event Hubs health check is disabled or not.
    /// </summary>
    /// <value>
    /// The default value is <see langword="false"/>.
    /// </value>
```

Reply:
I know it's a lame request but it makes everything the same. Will also require updating the schema files.